src/lib/index.jsx

```diff
@@ -213,6 +214,8 @@ class OtpInput extends Component<Props, State> {
     } else if (e.keyCode === RIGHT_ARROW || e.key === 'ArrowRight') {
       e.preventDefault();
       this.focusNextInput();
+    } else if (e.keyCode === SPACEBAR || e.key === 'Space' ) {
```

Comment:
Hey @utkarsh22garg!

`e.key` needs to be updated. It could be `" "` or on older browsers it's `'Spacebar'`

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values#Whitespace_keys

The rest looks good!

Reply:
@apollonian  I have made changes suggested by you. Kindly review it and let me know any changes.